CONTRIBUTING.md

```diff
@@ -96,6 +97,12 @@ When multiple versions coexist, **terms are only tracked in their English versio
 
 > We intend to expand coverage, but we focus for the moment on this subset of documents to fine-tune the system.
 
+### `executeClientScripts`
+
+This boolean property, when set to `true` executes client script before trying to get document content by browsing to the URL with an headless browser. It's set to `false` by default. It is useful when the content of the document is loaded or modified dynamically on the browser side by client scripts.
+
+**:warning: Browsing with an headless browser is expensive and time consuming, use only when necessary.**
```

Comment:
I'm not sure this is needed after the rewrite above. In case it is worth adding, see suggestion below 🙂 

```suggestion
**:warning: Fetching content with a headless browser is expensive and time consuming, use only when necessary.**
```